### PR TITLE
feat: added tron support

### PR DESCRIPTION
**Pull request by vrockz747**

### CODE REVIEW

File: packages/app-tron/src/utils/http.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import axios from 'axios';
```
Comment (irshadjsr21) [inline packages/app-tron/src/utils/http.ts:1]: Unused file

File: packages/app-tron/tests/01.create/index.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import { MockDeviceConnection } from '@cypherock/sdk-interfaces';
```
Comment (irshadjsr21) [inline packages/app-tron/tests/01.create/index.ts:1]: Need to add tests for all functions of `app-tron`

**Review by irshadjsr21 — CHANGES_REQUESTED** [review]

File: packages/app-tron/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# @cypherock/sdk-app-tron
```
[the remark this answers is not in the document]
Comment (vrockz747) [reply packages/app-tron/CHANGELOG.md]: removed

File: packages/app-tron/package.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@cypherock/sdk-app-tron",
+  "version": "0.0.5",
```
[the remark this answers is not in the document]
Comment (vrockz747) [reply packages/app-tron/package.json]: updated

File: packages/app-tron/src/utils/http.ts (hunk 1 above)
Comment (vrockz747) [reply packages/app-tron/src/utils/http.ts]: file removed

File: packages/app-tron/tests/01.create/index.ts (hunk 2 above)
Comment (vrockz747) [reply packages/app-tron/tests/01.create/index.ts:1]: test cases added

**Review by irshadjsr21 — APPROVED** [review]

**Pull request merged**